bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -100,15 +76,21 @@ impl RestApiConfigBuilder {
         self
     }
 
+    /// Sets the JWT salt.
+    pub fn jwt_salt(mut self, jwt_salt: String) -> Self {
```

Comment:
I'd call all these methods `with_<field>` or `set_<field>` instead, what do you think?

Reply:
Good point, added the `with` prefix